tso_session/ExploringTSOProducts.ipynb

```diff
@@ -114,15 +114,87 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    fig, ax1 = plt.subplots(figsize=(30, 5))
</pre><p>I found a figsize between (15, 5) and (20, 5) to be more readable across a variety of screen resolutions, but will leave the final choice up to you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='173'/>